scripts/yrd/install

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+# get previous location - should be 
+START_DIR="$(pwd)"
```

Comment:
I think this line should go after the `BASE_DIR` line.

Reply:
Don't think `START_DIR` is even needed. The place that it is used `BASE_DIR` should have been used for consistency